CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # [unreleased]
 ### Added
 - `python3.11-venv-d4tools` Dockerfile - not containing the pyd4 library
-- Use d4tools 0.3.10 in `python3.11-venv-d4tools` Dockerfile - install via git with tag
 ### Changed
 - Issue templates
+- Modified d4tools-based images to use GitHub branch master when cargo installs d4tools 
```

Comment:
This is because they are extremely slow to create new releases. Last time it took months after an important fix that was already merged in master branch